legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-dbExtension/legend-engine-xt-relationalStore-databricks/legend-engine-xt-relationalStore-databricks-pure/src/main/resources/core_relational_databricks/relational/protocols/pure/v1_23_0/extension/extension_databricks.pure

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Goldman Sachs
```

Comment:
v1_21, v1_22 missing?

Reply:
They are not present. They were probably not picked up when those protocols were cut